pkg/dns/dns.go

```diff
@@ -57,6 +58,9 @@ type KubeDNS struct {
 	// to get Endpoints and Service objects.
 	kubeClient clientset.Interface
 
+	// skydns points to the skydns server instance for configuration syncing.
+	skydnsConfig *server.Config
```

Comment:
Let's just make this a Public field and call set directly

`SkyDNSConfig *server.Config`

Reply:
Done.